activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -64,11 +64,15 @@ This is a list of questions for current owners of potential codebases to help us
 5. Is there a public bugtracking system in English or one you can make public?
 6. Is the codebase being deployed by another organization?
 7. Has any other organization expressed an interest in deploying the code?
+8. Is there a defined process for contributing to the code base?
```

Comment:
@zorp Is this significantly different from "7. Do you have contributor guidelines to help new people contribute?" under Community? Does it make more sense to you to have it here?

Reply:
True no reason to have it twice.

Reply:
Good, so we can remove this here, however, I like the way this is formulated (e.g. process versus just guidelines), so perhaps that can be added above.

```suggestion
```